### PR TITLE
Change project order by rating and budget

### PR DESCRIPTION
**Pull request by srisankethu**
Changes projectList order by 'rating' or 'budget', solves https://opengift.io/task_detail/?number=71&project=495

### CODE REVIEW

File: PManager/viewsExt/projects.py [hunk 1]
```diff
@@ -102,19 +103,34 @@ def recursiveTreeDraw(treeItem, depth=1):
 
     aProjects = []
     projects = PM_Project.objects.filter(public=True).order_by('-id')
```
Comment (gvammer) [inline PManager/viewsExt/projects.py]: here you should get projects only for the current page

Comment (gvammer) [inline PManager/viewsExt/projects.py]: you can use this https://docs.djangoproject.com/en/2.1/topics/pagination/

**Comment by srisankethu** [comment]
Closing this PR and opening a new one since it needs a fundamental change

File: PManager/viewsExt/projects.py [hunk 2]
```diff
@@ -102,19 +103,34 @@ def recursiveTreeDraw(treeItem, depth=1):
...
+    elif kwargs['order_by'] == 'budget':
+        aProjects = sorted(aProjects, key=lambda t: t.donated)
+
+    total_pages = int(ceil(float(len(aProjects))/page_size))
+
     c = RequestContext(request, {
         'specialties': aSpec,
         'spectree': recursiveTreeDraw({'subitems': aSpec.values()}),
-        'project_list': aProjects
+        'project_list': aProjects[(kwargs['page_no'] - 1)*page_size:min(kwargs['page_no']*page_size, len(aProjects))],
```
Comment (zodman) [inline PManager/viewsExt/projects.py]: i dont understand this line plz dont do more than 3 operations on one line

**Pull request closed without merging**